multimodal/src/autogluon/multimodal/matcher.py

```diff
@@ -1144,13 +1244,69 @@ def evaluate(
         else:
             return results
 
+    def evaluate(
+        self,
+        data: Optional[Union[pd.DataFrame, dict, list]] = None,
+        query_data: Optional[Union[pd.DataFrame, dict, list]] = None,
+        response_data: Optional[Union[pd.DataFrame, dict, list]] = None,
```

Comment:
Need to consider merge `response_data`, `query_data` and `data`.

Reply:
Per discussion, consider to merge "data" and "qr_relevance".

Reply:
Done. Removed `qr_relevance`.